nucypher/network/nodes.py

```diff
@@ -359,36 +357,35 @@ def load_seednodes(self, read_storage: bool = True, record_fleet_state=False):
                                                                        federated_only=self.federated_only,
                                                                        network_middleware=self.network_middleware,
                                                                        registry=self.registry)
-                except NodeSeemsToBeDown:
-                    self.unresponsive_seed_nodes.add(uri)
+                except Exception as e:
+                    # TODO: log traceback here?
+                    # TODO: distinguish between versioning errors and other errors?
+                    self.log.warn(f"Failed to instantiate a node at {uri}: {e}")
                 else:
-                    if maybe_sage_node is UNKNOWN_VERSION:
-                        continue
-                    else:
-                        new_node = self.remember_node(maybe_sage_node, record_fleet_state=False)
-                        discovered.append(new_node)
+                    new_node = self.remember_node(maybe_sage_node, record_fleet_state=False)
+                    discovered.append(new_node)
 
         for seednode_metadata in self._seed_nodes:
 
-            self.log.debug(
-                "Seeding from: {}|{}:{}".format(seednode_metadata.checksum_address,
-                                                seednode_metadata.rest_host,
-                                                seednode_metadata.rest_port))
-
-            seed_node = self.node_class.from_seednode_metadata(seednode_metadata=seednode_metadata,
-                                                               network_middleware=self.network_middleware,
-                                                               )
-            if seed_node is False:
-                self.unresponsive_seed_nodes.add(seednode_metadata)
-            elif seed_node is UNKNOWN_VERSION:
-                continue  # TODO: Bucket this?  We already emitted a warning.
+            node_tag = "{}|{}:{}".format(seednode_metadata.checksum_address,
+                                         seednode_metadata.rest_host,
+                                         seednode_metadata.rest_port)
+
+            self.log.debug(f"Seeding from: {node_tag}")
+
+            try:
+                seed_node = self.node_class.from_seednode_metadata(seednode_metadata=seednode_metadata,
+                                                                   network_middleware=self.network_middleware,
+                                                                   )
+            except Exception as e:
+                # TODO: log traceback here?
+                # TODO: distinguish between versioning errors and other errors?
```

Comment:
This is essentially the purpose of the `NodeSeemsToBeDown` tuple, otherwise this may be overtly generic.  I do think it's a good idea to have this catch-all for `Exception`.

Reply:
We could check for that first specifically, but the action would be the same - just log it and move on. So I am not sure it is necessary at this stage. Maybe if we bucketed the node behavior, there would be a difference between an unresponsive node, a node with incompatible version, and a node returning some trash.